src/marqo/tensor_search/models/api_models.py

```diff
@@ -67,16 +69,35 @@ def validate_search_method(cls, value):
             value=value, enum_class=SearchMethod,
             case_sensitive=False
         )
-    
+
     def get_context_tensor(self) -> Optional[List[SearchContextTensor]]:
         """Extract the tensor from the context, if provided"""
         return self.context.tensor if self.context is not None else None
 
+
+class RecommendQuery(BaseMarqoModel):
```

Comment:
It would be more uniform to move this class to its own module here: https://github.com/marqo-ai/marqo/tree/mainline/src/marqo/api/models